### PR TITLE
Improve sanitization of broken XML processing instructions during parsing

### PR DESCRIPTION
**Pull request by jbaiter**

### CODE REVIEW

File: example/frontend/Dockerfile [hunk 1]
```diff
@@ -11,6 +11,7 @@ RUN apk add git &&\
 FROM nginx:alpine
 
 COPY --from=builder /build/build /usr/share/nginx/html
+RUN rm -rf /usr/share/nginx/html/sw.js
```
Comment (schmika) [inline example/frontend/Dockerfile:14]: Why is this necessary? Hasn't this already been removed in line 9?

Comment (jbaiter) [reply example/frontend/Dockerfile:14]: It has indeed and it's not necesary, thank you!

**Review by schmika — APPROVED** [review]
